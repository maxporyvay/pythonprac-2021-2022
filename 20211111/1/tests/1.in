A, B = OrderedComplex(1, -5), OrderedComplexMul(10, 15)
print(A<B, B@B)
