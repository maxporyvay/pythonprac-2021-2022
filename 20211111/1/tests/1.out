True 325.0
